hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -72,60 +95,54 @@ class ReadOnlyHandle
 
   double get_value() const
   {
+    // BEGIN (Handle export change): for backward compatibility
+    // TODO(Manuel) return value_ if old functionality is removed
     THROW_ON_NULLPTR(value_ptr_);
     return *value_ptr_;
+    // END
+  }
+
+  void set_value(double value)
+  {
+    // BEGIN (Handle export change): for backward compatibility
+    // TODO(Manuel) set value_ directly if old functionality is removed
+    THROW_ON_NULLPTR(this->value_ptr_);
+    *this->value_ptr_ = value;
+    // END
```

Comment:
I've a question here. Now everyone who has access to the `StateInterfaceHandle` will also be able to set the value right?. Shouldn't we add this `set_value` method only to the `CommandInterfaceHandle`?

Reply:
Sure. Got it. 
Thanks for explaining it. Is this restriction at the loaned interfaces level already applied to the code in this PR?

Reply:
This restriction is in the nature of the Loaned Interface, as we are using there already getter and setters and we should be adding additional features for this not to be availble.

See the files here that are unchanged:
- https://github.com/StoglRobotics-forks/ros2_control/blob/change_interface_export_variant/hardware_interface/include/hardware_interface/loaned_command_interface.hpp
- https://github.com/StoglRobotics-forks/ros2_control/blob/change_interface_export_variant/hardware_interface/include/hardware_interface/loaned_state_interface.hpp